src/lib.rs

```diff
@@ -12,7 +12,7 @@ use services::config::manager::ConfigManager;
 use services::error::IoError;
 use services::statefuls::cache::manager::CacheManager;
 use services::statefuls::cache::ttl::manager::TtlSchedulerInbox;
-use services::statefuls::persist::actor::PersistActor;
+use services::statefuls::snapshot::dump_loader::DumpLoader;
```

Comment:
let's make it consistent, if we are to rename it as snapshot, let's call it snapshot lodaer